app/views/users/index.html.erb

```diff
@@ -1 +1,18 @@
-<h1>All the users</h1>
\ No newline at end of file
+<section class="p-sm mb-section">
+  <ul class="container-user d-flex-2">
+    <% @users.each do |user| %>
+      <li class="user-li d-flex border">
+        <div class="img-con">
+          <img class="av" src="<%= user.photo %>" alt="User Avatar" />
+        </div>
+
+        <div class="user-bdy d-flex">
+          <a class="user-pc d-flex" title="view user details" href="/users/<%= user.id %>">
+            <h2 class="fs-sm"><%= user.name%></h2>
+            <p class="post-count">Number of posts: <%= user.posts_counter || 0 %> </p>
+          </a>
+        </div>
+      </li>
+    <% end %>
```

Comment:
- [OPTIONAL] All the pages except the **_Post show page_** use the same layout to display the **_user's information_**. It would be nice to use a `partial` to render the **_user's information_** instead of repeating the same code in each file. You can read this article to learn more about using [partials](https://rails.devcamp.com/trails/learn-ruby-on-rails-from-scratch/campsites/configuring-rails-views/guides/how-to-use-partials-in-rails)